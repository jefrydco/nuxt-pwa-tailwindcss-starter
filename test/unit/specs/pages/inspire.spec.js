import { shallowMount, createLocalVue, RouterLinkStub } from '@vue/test-utils'

import Inspire from '~/pages/inspire'
import { div } from '~/test/utils'

const localVue = createLocalVue()

describe('Inspire', () => {
  it('renders to match snapshot', () => {
    const wrapper = shallowMount(Inspire, {
      localVue,
      stubs: {
        'nuxt-link': RouterLinkStub,
        'nuxt-child': div
      }
    })
    expect(wrapper).toMatchSnapshot()
  })
})
