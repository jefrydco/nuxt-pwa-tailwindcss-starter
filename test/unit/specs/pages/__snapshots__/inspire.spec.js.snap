// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Inspire renders to match snapshot 1`] = `
<div>
  <h1>Inspire</h1> <a class="px-2 -mx-2 py-1 transition duration-200 ease-in-out relative block text-teal-600 font-medium underline">
    Home
  </a>
</div>
`;
